js.go

```diff
@@ -2575,6 +2575,12 @@ func checkMsg(msg *Msg, checkSts, isNoWait bool) (usrMsg bool, err error) {
 			// one message when making requests without no_wait.
 			err = ErrTimeout
 		}
+	case jetStream409Sts:
+		if strings.Contains(strings.ToLower(string(msg.Header.Get(descrHdr))), "consumer deleted") {
```

Comment:
so how are we handling this client facing breaking change?

Reply:
This status was introduced last week in the server (notifying about consumer being deleted) and we only depend on "consumer deleted" string, so think we can treat this as a new feature: https://github.com/nats-io/nats-server/pull/3605

WDYT?

Reply:
I meantion the nats.go release being 3 days ago to highlight that the next nats.go release is probably not tomorrow - meaning the delta between current users on a current server and this change langing in a release is likely to be quite big, relatively speaking.  Thus increasing the risk that this will impact real users.

Reply:
You're right about that. To shorten the delta we could issue a patch release sooner (e.g. tomorrow), minimizing the impact.